bpf/lib/encap.h

```diff
@@ -145,10 +151,11 @@ __encap_with_nodeid(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
 
 static __always_inline int
 __encap_and_redirect_with_nodeid(struct __ctx_buff *ctx, __u32 tunnel_endpoint,
-				 __u32 seclabel, const struct trace_ctx *trace)
+				 __u32 seclabel, __u32 vni __maybe_unused,
```

Comment:
Shouldn't need `__maybe_unused` if the function passes the value down to another function?
```suggestion
				 __u32 seclabel, __u32 vni,
```

Reply:
(Only applicable here, the other one _does_ need __maybe_unused since the variable is only used in a conditional macro section).